AzureCalling/app/src/main/res/layout-land/activity_call.xml

```diff
@@ -31,27 +31,32 @@
         app:layout_constraintLeft_toLeftOf="parent"
         app:layout_constraintTop_toTopOf="parent">
 
-        <GridLayout
-            android:id="@+id/groupCallTable"
-            android:layout_width="0dp"
-            android:layout_height="0dp"
-            android:clickable="true"
-            app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintEnd_toEndOf="parent"
-            app:layout_constraintTop_toTopOf="parent"
-            app:layout_constraintBottom_toBottomOf="parent"/>
-
-        <include layout="@layout/floating_header_area" />
-
         <androidx.constraintlayout.widget.ConstraintLayout
-            android:id="@+id/yourCameraHolder"
-            android:layout_width="120dp"
-            android:layout_height="70dp"
-            android:layout_marginEnd="20dp"
-            android:layout_marginBottom="20dp"
-            app:layout_constraintBottom_toBottomOf="parent"
-            app:layout_constraintEnd_toEndOf="parent">
-
+            android:id="@+id/relativeLayout"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent">
+
+            <include layout="@layout/participant_header_notification"/>
+            <include layout="@layout/floating_header_area" />
```

Comment:
Looks like this element no longer shows up during Teams calls.  Specifically, the recording notifications.